test/uk/gov/hmrc/pillar2/connectors/SubscriptionConnectorSpec.scala

```diff
@@ -31,7 +35,7 @@ class SubscriptionConnectorSpec extends BaseSpec with Generators with ScalaCheck
       "microservice.services.create-subscription.port" -> server.port()
     )
     .build()
-
+  private val errorCodes: Gen[Int] = Gen.oneOf(Seq(203, 204, 400, 403, 500, 501, 502, 503, 504))
```

Comment:
Do we definitely want to treat 204 and 203 as error codes? Also this function could probably go in a util/helper class.

Reply:
This value is not universal across all classes so it can't be generalised. And yes, we need to treat anything non-OK as an error code for amendSubscription based on the API spec

Reply:
Ah okay its based off the API spec. I understand.